app.js

```diff
@@ -72,11 +78,25 @@ io.on('connection', (socket) => {
     // import map class
     // eslint-disable-next-line global-require
     const { GameMap } = require('./public/models/GameMap');
-    const mapFile = `./private/assets/maps/map${Utils.getRandomNumber(1, 6)}.json`;
+    //const mapFile = `./private/assets/maps/map${Utils.getRandomNumber(1, 6)}.json`;
+    const mapFile = `./private/assets/maps/map0.json`;
     const map = new GameMap('', '', '', '', '');
     // eslint-disable-next-line global-require
     map.loadMap(mapFile); // eslint-disable-line import/no-dynamic-require
     games[roomName].map = map;
+    //get logged in player id
+    let id = '';
+    let response = await fetch(`${process.env.URL}getsession`);
+    let result = await response.json();
+    if (result.playerId) {
+      id = result.playerId;
+    }
+    //load player interactions
+    let url = `${process.env.URL}api/interaction`;
+    if (id) url += `?player_id=${id}`;
+    response = await fetch(url);
+    result = await response.json();
+    games[roomName].interactions = result;
```

Comment:
you just added the interactions to the game objects?

Reply:
yes so it just gets loaded once when the game starts and then players choose random values from there so we don't need to query database all the time